apps/fis/fis-cos-api/demo.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   values:
     java:
-      image: hmctspublic.azurecr.io/fis/cos-api:pr-200-675c599-20240116170953 #{"$imagepolicy": "flux-system:demo-fis-cos-api"}
+      image: hmctspublic.azurecr.io/fis/cos-api:pr-263-6b44d69-20240904152211 #{"$imagepolicy": "flux-system:fis-cos-api-demo"}
```

Comment:
?

Wha'ts the aim with this - i don't see that fis-cos-api-demo image policy, did you mean for `fis-cos-api`